README.md

```diff
@@ -123,6 +123,8 @@ The following table lists all valid connection properties:
 | USER                       | Yes      | If AUTHENTICATOR is set to `externalbrowser` or the URL for native SSO through Okta, set this to the login name for your identity provider (IdP).     |
 | WAREHOUSE                  | No       |                                                                               |
 | CONNECTION_TIMEOUT         | No       | Total timeout in seconds when connecting to Snowflake. The default is 120 seconds |
+| BROWSER_RESPONSE_TIMEOUT   | No       | Number to seconds to wait for authentication in an external browser (default: 120). |
+| MAXHTTPRETRIES             | No       | Maximum number of times to retry a failed HTTP requests (default: 7). You can can set `MAXHTTPRETRIES=0` to remove the retry limit, but doing so runs the risk of the .NET driver infinitely retrying failed HTTP calls. |
```

Comment:
two typos:

- "a failed HTTP requests" - singular or plural?
- "You can can set..."

Reply:
Fixed